### ISSUE TITLE
Contributions

### ISSUE THREAD

**Issue by jackmcpickle** [jasalo/funcssion#2]
Hey @jasalo, 

really like the idea behind this project - I had planned to build something very similar. 😄 

Try to get the repo running locally. But there are n build methods (gulp/node-sass/ruby-sass) defined. Is this deliberate or are you planning on adding a package.json later. 

Otherwise happy to create one?

**Comment by jasalo** [comment]
Yeah! I was using [HarpJS](https://harpjs.com/) to quickly compile SASS and Jade files and just focus on development initially. Now that v0.1 is out, I'm planning to add a `package.json` and use webpack to handle SASS, Jade compilation. I'll work on that soon, so I'll keep you posted :) Thanks for the interest!

**Comment by jackmcpickle** [comment]
Check out https://github.com/oddbird/true too. Ive used it before to write sass tests for my projects. Makes refactoring easier. 

And maybe a linter for sass too? like sass-lint or style-lint.
